### PR TITLE
Update Symfony, fix Demo application

### LINKED ISSUE #60: Edit Purchases Or User leads to 500
Hi,
i installed the project and follow the README.md, except that i use default doctrine param with the sqlite database.
The app is running on localhost:8001
If i try to edit one purchase then it fail and crash on 500

Go on a Purchase detail : http://localhost:8001/admin/?action=show&entity=Purchase&menuIndex=2&submenuIndex=-1&sortField=deliveryDate&sortDirection=DESC&page=1&referer=%252Fadmin%252F%253Faction%253Dlist%2526entity%253DPurchase%2526menuIndex%253D2%2526submenuIndex%253D-1%2526sortField%253DdeliveryDate%2526sortDirection%253DDESC%2526page%253D1&id=120180030803276001515076665

Then try to edit it : http://localhost:8001/admin/?entity=Purchase&referer=%252Fadmin%252F%253Faction%253Dlist%2526entity%253DPurchase%2526menuIndex%253D2%2526submenuIndex%253D-1%2526sortField%253DdeliveryDate%2526sortDirection%253DDESC%2526page%253D1&action=edit&id=120180030803276001515076665

Expect : the form to edit the purchase

Get : an HTTP 500

You can repeat this process with a User edit.

### LINKED ISSUE #69: Demo does not work at all Oops! An Error Occurred
Experienced user will not need a demo - is this agreeable? 

So you should expect newbies to try this - and get frustrated hitting several walls.

I tried to understand why there is no `app/app/config/parameters.yml` and found 
`config/packages/easy_admin.yaml` - however after following the instructions and visiting `127.0.0.1:8000/admin` I get an error:

```
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused. 
```

The dev webserver logs this: 
```
[Sun Dec 23 21:03:21 2018] 127.0.0.1:56598 [500]: / - Uncaught Symfony\Component\Config\Definition\Exception\InvalidTypeException: The configured class "AppBundle\Entity\Category" for the path "easy_admin.entities.Category2" does not exist. Did you forget to create the entity class or to define its namespace? in /path/to/symfony/easy-admin-demo/vendor/javiereguiluz/easyadmin-bundle/src/Configuration/MetadataConfigPass.php:40
Stack trace:
#0 /path/to/symfony/easy-admin-demo/vendor/javiereguiluz/easyadmin-bundle/src/Configuration/ConfigManager.php(204): EasyCorp\Bundle\EasyAdminBundle\Configuration\MetadataConfigPass->process(Array)
#1 /path/to/symfony/easy-admin-demo/vendor/javiereguiluz/easyadmin-bundle/src/Configuration/ConfigManager.php(187): EasyCorp\Bundle\EasyAdminBundle\Configuration\ConfigManager->doProcessConfig(Array)
#2 /path/to/symfony/easy-admin-demo/vendor/javiereguiluz/easyadmin-bundle/src/Configuration/ConfigManager.php(67): EasyCorp\Bundle\EasyAdminBundle\Configuration\ConfigManager->processConfig()
#3 /path/to/symfo in //path/to/symfony/easy-admin-demo/vendor/javiereguiluz/easyadmin-bundle/src/Configuration/MetadataConfigPass.php on line 40
[Sun Dec 23 21:03:23 2018] 127.0.0.1:56600 [301]: /
[Sun Dec 23 21:03:23 2018] 127.0.0.1:56602 [500]: /admin/
[Sun Dec 23 21:03:31 2018] 127.0.0.1:56604 [500]: /admin/
[Sun Dec 23 21:03:33 2018] 127.0.0.1:56606 [500]: /admin/
[Sun Dec 23 21:03:34 2018] 127.0.0.1:56608 [500]: /admin/
[Sun Dec 23 21:30:03 2018] 127.0.0.1:56814 [500]: /admin/
```
The line that starts with `#3` is no copy-paste error - the first path just ends with `symfo`.

### PR DESCRIPTION
**Pull request by IonBazan**
This makes the Demo App fully working and upgrades to most recent versions of Symfony, EasyAdminBundle, VichUploaderBundle and replaces IvoryCKEditorBundle with FOSCKEditorBundle

This should fix #69 and EasyCorp/EasyAdminBundle#2554 issues.
EDIT: Also fixes #60 by forcing `texarea` form type and #66 by bumping minimal PHP version to 7.2.9 (as Symfony 5.0).

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Great 👏  Thanks a lot! I've installed and it looks nice. I've only seen some very minor issues in some pages. I'll review with more detail later!

**Comment by IonBazan** [comment]
@javiereguiluz Thanks. I think we should also move this repository to @EasyCorp. WDYT?
A working online demo to show some cool features of the bundle would also be appreciated.

File: .env.test [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# define your env variables for the test env here
+KERNEL_CLASS='App\Kernel'
+APP_SECRET='s$cretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=max[self]=0
```
Comment (IonBazan) [inline .env.test:4]: This is to ignore deprecations introduced by VichUploaderBundle.

**Comment by javiereguiluz** [comment]
Ion, thanks again for your great work here. I wanted to do this so bad ... but I never found the time for it. It's merged now!

**Comment by IonBazan** [comment]
I am glad I was able to help here :)

**Pull request merged**